wicker/schema/schema.py

```diff
@@ -507,6 +606,10 @@ def process_double_field(self, field: DoubleField) -> _T:
     def process_object_field(self, field: ObjectField) -> _T:
         pass
 
+    @abc.abstractmethod
+    def process_sf_variant_field(self, field: ObjectField) -> _T:
+        pass
+
```

Comment:
Is it better to add a function to handle new field type? Or better to expand existing function to handle `str` instance?